android/src/com/mapswithme/maps/editor/EditorFragment.java

```diff
@@ -499,29 +495,43 @@ public void onClick(View v)
     }
   }
 
-  private void refreshLocalizedNames()
+  private void showAdditionalNames(boolean show)
   {
-    UiUtils.showIf(mLocalizedNamesAdapter.getItemCount() > 0, mLocalizedShow);
+    mNamesAdapter.showAdditionalLanguages(show);
+
+    refreshNamesCaption();
   }
 
-  private void showLocalizedNames(boolean show)
+  private void refreshNamesCaption()
   {
-    mIsLocalizedShown = show;
-    if (show)
-    {
-      UiUtils.show(mLocalizedNames);
-      setLocalizedShowDrawable(R.drawable.ic_expand_less);
-    }
+    if (mNamesAdapter.getNamesCount() <= mNamesAdapter.getMandatoryNamesCount())
+      setNamesArrow(0 /*arrowResourceId*/);  // Bind arrow with empty resource (do not draw arrow).
```

Comment:
```java
/* arrowResourceId */
```
I.e. surround comment by spaces please.
Also, it's optional, but inline-comments are usually written in the following format:
```java
// bind arrow with empty resource (do not draw arrow)
```
I.e. as incomplete sentence.